src/feature-test/java/uk/gov/ida/verifyserviceprovider/configuration/VerifyHubConfigurationFeatureTests.java

```diff
@@ -1,37 +1,50 @@
-package feature.uk.gov.ida.verifyserviceprovider.configuration;
+package uk.gov.ida.verifyserviceprovider.configuration;
 
 import certificates.values.CACertificates;
 import keystore.KeyStoreResource;
-import keystore.KeyStoreRule;
 import keystore.builders.KeyStoreResourceBuilder;
+import org.junit.After;
 import org.junit.Before;
 import org.junit.ClassRule;
 import org.junit.Test;
-import uk.gov.ida.verifyserviceprovider.configuration.HubMetadataConfiguration;
-import uk.gov.ida.verifyserviceprovider.configuration.VerifyHubConfiguration;
+import org.junit.contrib.java.lang.system.EnvironmentVariables;
+import uk.gov.ida.saml.metadata.MetadataResolverConfiguration;
 
 import static org.assertj.core.api.Java6Assertions.assertThat;
+import static uk.gov.ida.verifyserviceprovider.configuration.ConfigurationConstants.EnvironmentVariables.HUB_EXPECTED_ENTITY_ID;
+import static uk.gov.ida.verifyserviceprovider.configuration.ConfigurationConstants.EnvironmentVariables.HUB_METADATA_URL;
+import static uk.gov.ida.verifyserviceprovider.configuration.ConfigurationConstants.EnvironmentVariables.HUB_SSO_URL;
+import static uk.gov.ida.verifyserviceprovider.configuration.ConfigurationConstants.EnvironmentVariables.HUB_TRUSTSTORE_PATH;
+import static uk.gov.ida.verifyserviceprovider.configuration.ConfigurationConstants.EnvironmentVariables.IDP_TRUSTSTORE_PATH;
+import static uk.gov.ida.verifyserviceprovider.configuration.ConfigurationConstants.EnvironmentVariables.METADATA_TRUSTSTORE_PATH;
+import static uk.gov.ida.verifyserviceprovider.configuration.ConfigurationConstants.EnvironmentVariables.TRUSTSTORE_PASSWORD;
 import static uk.gov.ida.verifyserviceprovider.configuration.ConfigurationConstants.HUB_JERSEY_CLIENT_NAME;
-import static uk.gov.ida.verifyserviceprovider.configuration.ConfigurationConstants.PRODUCTION_VERIFY_TRUSTSTORE_NAME;
-import static uk.gov.ida.verifyserviceprovider.configuration.ConfigurationConstants.TEST_VERIFY_TRUSTSTORE_NAME;
 import static uk.gov.ida.verifyserviceprovider.utils.DefaultObjectMapper.OBJECT_MAPPER;
 
 public class VerifyHubConfigurationFeatureTests {
 
     private static final KeyStoreResource keyStore = KeyStoreResourceBuilder.aKeyStoreResource()
```

Comment:
Can we replace `keyStore` with `KEYSTORE` since this variable is using both `static` and `final`.